engine/src/main/java/org/camunda/bpm/engine/impl/telemetry/TelemetryLogger.java

```diff
@@ -26,59 +27,85 @@ public void startTelemetrySendingTask() {
   }
 
   public void exceptionWhileSendingTelemetryData(Exception e) {
+    logWarn("002",
+        "Could not send telemetry data. Reason: {} with message '{}'. Set this logger to DEBUG/FINE for the full stacktrace.",
+        e.getClass().getSimpleName(),
+        e.getMessage());
     logDebug(
-        "002", "'{}' exception occurred while sending telemetry data with message: {}", 
+        "003", "{} occurred while sending telemetry data.",
         e.getClass().getCanonicalName(),
-        e.getMessage());
+        e);
   }
 
-  public void unexpectedResponseWhileSendingTelemetryData(int responseCode) {
-    logDebug(
-        "003", "Unexpected response while sending telemetry data. Status code: {}", responseCode);
+  public ProcessEngineException unexpectedResponseWhileSendingTelemetryData(int responseCode) {
+    return new ProcessEngineException(
+      exceptionMessage("013", "Unexpected response code {} when sending telemetry data", responseCode));
```

Comment:
maybe this id should be 004 instead?
```suggestion
      exceptionMessage("004", "Unexpected response code {} when sending telemetry data", responseCode));
```